runtime/rasdump/dmpsup.c

```diff
@@ -577,7 +577,42 @@ configureDumpAgents(J9JavaVM *vm)
 		}
 		if ( IS_CONSUMABLE(j9vm_args, xdumpIndex) && !IS_CONSUMED(j9vm_args, xdumpIndex) )
 		{
-			GET_OPTION_VALUE(xdumpIndex, ':', &optionString);
+			BOOLEAN isMappedToolDump = FALSE;
+			/* Handle mapped tool dump options */
+			if (HAS_MAPPING(j9vm_args, xdumpIndex)) {
+				char *mappingJ9Name = MAPPING_J9NAME(j9vm_args, xdumpIndex);
+				char *toolString = ":tool:";
+				char *toolCursor = strstr(mappingJ9Name, toolString);
+				if (NULL != toolCursor) {
+					char *optionValue = NULL;
+					/* The mapped option specifies the tool command to run after the equals */
+					GET_OPTION_VALUE(xdumpIndex, '=', &optionValue);
+					
+					/* Move toolCursor past ":tool:" */
+					toolCursor += strlen(toolString) + 1;
```

Comment:
I don't think you want the `+ 1` here: it would have the effect of pointing at `vents=systhrow...` in `-Xdump:tool:events=systhrow...`.